catalystwan/api/administration.py

```diff
@@ -408,7 +408,7 @@ def __update_password(self, payload: dict) -> Response:
     def __update_certificate(self, payload: Certificate) -> Response:
         json_payload = asdict(payload)  # type: ignore
         endpoint = "/dataservice/settings/configuration/certificate"
-        return self.session.put(endpoint, json=json_payload)
+        return self.session.post(endpoint, json=json_payload)
```

Comment:
If we have endpoint maybe use it here?
```suggestion
        return self.session.endpoints.configuration_settings.edit_certificates(payload)
```